tools/c7n_gcp/c7n_gcp/policy.py

```diff
@@ -170,3 +170,94 @@ def run(self, event, context):
                 action.process(resources)
 
         return resources
+
+
+@execution.register('gcp-scc')
+class SCCMode(FunctionMode):
+    """Custodian policy execution on gcp scc findings.
+
+    Deploys as a Cloud Function triggered by scc findings. This allows
```

Comment:
nit/docs.. we probably need at least one expanded ref to Security Command Center for folks unfamiliar with the scc abbreviation.